sdk/src/wallet/account/operations/syncing/addresses/output_ids/mod.rs

```diff
@@ -44,6 +44,38 @@ where
             return Ok(output_ids);
         }
 
+        // If interested in alias, basic and NFT outputs, get them all at once
+        if (address.is_ed25519()
+            && sync_options.account.alias_outputs
+            && sync_options.account.basic_outputs
+            && sync_options.account.nft_outputs)
```

Comment:
May an `all_outputs` method could be handy

Reply:
But it's only here and for aliases foundry outputs are not needed here

Reply:
Still, DRY

Reply:
What should be the name for the alias sync options then? `all_outputs_except_foundry()` or what?

Reply:
Good enough to start I think

Reply:
I initially didn't see `AliasSyncOptions` has the foundry one.  I would maybe add an all method to all of them, that actually does an all. But not use it for alias. Bit weird to have a all_outputs_except_foundry I think.

Reply:
Don't see a use for all for the AliasSyncOptions currently and since it's just internal I didn't add it